<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addDevice</class>
 <widget class="QWidget" name="addDevice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="deleteBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>310</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="alterBtn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QToolButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="resoure.qrc">
     <normaloff>:/image/close.png</normaloff>:/image/close.png</iconset>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>381</width>
     <height>241</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="usesScrollButtons">
    <bool>true</bool>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="chest">
    <attribute name="title">
     <string>机柜</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">font: 75 10pt &quot;Verdana&quot;;</string>
          </property>
          <property name="text">
           <string>型号：      </string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Verdana&quot;;</string>
          </property>
          <property name="text">
           <string>厂家：      </string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Verdana&quot;;</string>
          </property>
          <property name="text">
           <string>名称：      </string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Verdana&quot;;</string>
          </property>
          <property name="text">
           <string>高度(u)：</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QLineEdit" name="chestFactoryEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLineEdit" name="chestVersionEdit"/>
        </item>
        <item row="3" column="0">
         <widget class="QLineEdit" name="chestHeightEdit">
          <property name="text">
           <string>42</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="chestNameEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="chestInfoLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>351</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>chestInfoLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="iscan">
    <attribute name="title">
     <string>扫描仪</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="1">
       <widget class="QLineEdit" name="iscanNameEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Verdana&quot;;</string>
        </property>
        <property name="text">
         <string>名称：      </string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_10">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Verdana&quot;;</string>
        </property>
        <property name="text">
         <string>网关：      </string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="iscanGatewayEdit">
        <property name="text">
         <string>192.168..</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="iscanIpEdit">
        <property name="text">
         <string>192.168..</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Verdana&quot;;</string>
        </property>
        <property name="text">
         <string>IP：      </string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="iscanPortEdit">
        <property name="text">
         <string>20012</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_11">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Verdana&quot;;</string>
        </property>
        <property name="text">
         <string>端口：      </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="iscanInfoLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>187</y>
       <width>351</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>iscanInfoLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="switch_2">
    <attribute name="title">
     <string>交换机</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="3" column="2">
       <widget class="QLineEdit" name="switchModel"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_18">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Verdana&quot;;</string>
        </property>
        <property name="text">
         <string>型号：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_19">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Verdana&quot;;</string>
        </property>
        <property name="text">
         <string>厂家：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="switchManufactory"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="switchNameEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_20">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Verdana&quot;;</string>
        </property>
        <property name="text">
         <string>名称：      </string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>端口数：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="switchPortEdit">
        <property name="text">
         <string>24</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="switchInfoLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>351</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>switchInfoLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Peixianjia">
    <attribute name="title">
     <string>电子配线架</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="3" column="0">
       <widget class="QLabel" name="label_15">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Verdana&quot;;</string>
        </property>
        <property name="text">
         <string>型号：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="peiManufactory"/>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="peiModel"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_14">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Verdana&quot;;</string>
        </property>
        <property name="text">
         <string>厂家：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Verdana&quot;;</string>
        </property>
        <property name="text">
         <string>名称：      </string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="peiNameEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_13">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Verdana&quot;;</string>
        </property>
        <property name="text">
         <string>类别：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="peiComboBox"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="peiInfoLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>351</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>peiInfoLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Verdana&quot;;</string>
   </property>
   <property name="text">
    <string>添加设备</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resoure.qrc"/>
 </resources>
 <connections/>
</ui>
